gulpfile.js

```diff
@@ -283,7 +283,7 @@ gulp.task('run', ['clean', 'serve'], function() {
   gulp.watch(config.folderAssets.fonts + '/*.*', ['copy:fonts', reload]);
   gulp.watch(config.folderAssets.images + '/**/*.*', ['copy:images']);
   gulp.watch(config.folderAssets.js + '/**/*.js', ['copy:js', reload]);
-  gulp.watch(config.folderAssets.base + '/templates/*.html', ['processHtml']);
+  gulp.watch(config.folderAssets.base + '/templates/**/*.*', ['processHtml']);
```

Comment:
its right this change? no need process only html?

Reply:
We only have HTML files in that folder, and processHtml takes care of processing `.html` files only. What I've changed here is the `watch` task so each time *any file* changes on the `/templates` folder, processHtml is run.